Proposals/xgov-20.md

```diff
@@ -0,0 +1,199 @@
+---
+title: Building the Future of Competitive Gaming Grant Proposal for Our Free-to-Play Blockchain Game
+author: Aquiles benitez (@aquilesdel90) <benitez.aquiles@gmail.com>
+company_name: "Gunny games studios"
+category: Community
+focus_area: Gaming
+open_source:  No
+amount_requested: 833000
+status: Draft
```

Comment:
Remember to change this to "Final" if you want your proposal to go through the next voting session.

Reply:
Ready